portfolio/src/main/webapp/index.html

```diff
@@ -85,18 +85,21 @@ <h1>Adam's Personal Portfolio</h1>
       <div id="comment">
         <h2>Leave a Comment</h2>
         <form action="/data" method="POST">
-        
-        <label for="user-name">Your name:</label></br>
-        <input type="text" name="user-name">
-        </br></br>
 
-        <!--change format for comment box-->
-        <label for="user-comment">Your comment:</label></br>
-        <input type="text" name"user-comment">
+         <label for="user-name">Your name:</label></br>
+         <input type="text" name="user-name" required>
+         </br></br>
+
+         <!--change format for comment box-->
+         <label for="user-comment">Your comment:</label></br>
+         <input type="text" name="user-comment" required>
+         
+         <input type="submit"/>
+
         </form>
-      </div>
 
       <button onclick="getData()">See Comments</button>
```

Comment:
I think you mentioned that you're already thinking about this, but I found it a little confusing that I needed to click submit first, then 'See Comments' to view comments on the page. If you're planning to address that in a later CL then feel free to disregard this comment. otherwise maybe there's a way to get both to happen when you click submit (eg: submit my comment and show others with a single click).